app/models/manager_refresh/target.rb

```diff
@@ -46,6 +49,15 @@ def dump
     alias id dump
     alias name manager_ref
 
+    def payload=(data)
+      payload_blob.try(:destroy)
+      @payload_id = data ? BinaryBlob.create!(:binary => data).id : nil
```

Comment:
would it help giving the BB a name/data_type?

Reply:
Yeah that's a good idea

Reply:
Added a name of manager_id + __ + manager_ref, data_type is going to be harder because that's on the caller passing the data in and I don't want to just assume yaml/json based on what we're doing here